protocol/messenger_peersyncing.go

```diff
@@ -133,22 +151,22 @@ func (m *Messenger) sendDatasyncOffers() error {
 			continue
 		}
 
-		availableMessages, err := m.peersyncing.AvailableMessagesByGroupIDs(chatIDs, maxAdvertiseMessages)
+		availableMessages, err := m.peersyncing.AvailableMessagesByChatIDs(chatIDs, maxAdvertiseMessages)
```

Comment:
We seem to be constructing a chatID->msgID map in all cases when we invoke `m.peersyncing.AvailableMessagesByChatIDs` - maybe makes sense to make the latter fn return that map instead. (unless i missed some other uses)